### PR TITLE
Add includecheck leg

### PR DESCRIPTION
**Pull request by dcristoloveanu**

### CODE REVIEW

**Review by jebrando — APPROVED** [review]
:shipit:

File: src/umocktypes_wcharptr.c [hunk 1]
```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#include <errno.h>
 #include <string.h>
 #include <stddef.h>
 #include <stdlib.h>
 #include <wchar.h>
+#include <errno.h> // IWYU pragma: keep
```
Comment (mattdurak) [inline src/umocktypes_wcharptr.c:8]: >keep [](start = 35, length = 4)

Is there a way to put this in the config instead? Like make errno a symbol:
`{ symbol: ["errno", "private", "<errno.h>", "public"] },`
Just checking

**Review by mattdurak — APPROVED** [review]
:shipit:

File: src/umocktypes_wcharptr.c (hunk 1 above)
Comment (dcristoloveanu) [reply src/umocktypes_wcharptr.c:8]: This is an oddness where iwyu thinks errno is not needed, but actually it is. The only way to fix it is to inform iwyu that it is wrong :-). By a pragma. The rules mapping file that iwyu supports does not allow for expressing a "keep".

**Pull request merged**